tesseract_core/runtime/cli.py

```diff
@@ -467,41 +465,26 @@ def openapi_schema() -> dict:
     return group
 
 
-@contextlib.contextmanager
-def stdout_to_stderr() -> Generator:
-    """Redirect stdout to stderr at OS level."""
-    orig_stdout = os.dup(sys.stdout.fileno())
-    sys.stdout.flush()
-    os.dup2(sys.stderr.fileno(), sys.stdout.fileno())
-    try:
-        yield os.fdopen(orig_stdout, "w", closefd=False)
-    finally:
-        sys.stdout.flush()
-        os.dup2(orig_stdout, sys.stdout.fileno())
-
-
 def main() -> None:
     """Entrypoint for the command line interface."""
-    # Redirect stdout to stderr to avoid mixing any output with the JSON response.
-    with stdout_to_stderr() as orig_stdout:
```

Comment:
We should keep something like that to avoid stuff the runtime prints (e.g. a Python warning) from polluting return data. But it's OK to finish the rest of the PR first, then we can do it together (may be a bit finicky).